src/index.html

```diff
@@ -18,7 +18,7 @@
             // We need the RAW file from the root dir
             // https://github.com/parcel-bundler/parcel/issues/1087#issuecomment-576810737
             const script = document.createElement("script");
-            script.setAttribute("src", "env.js");
+            script.setAttribute("src", "/env.js");
```

Comment:
would this make problems when hosting in a subpath

Reply:
Yes. The env.js is placed on the same location like the editor base stuff. But the urls /course/id/... also include in the path if they load the env without /.